ui/packages/ui/src/app/pages/createConnector/CreateConnectorPage.tsx

```diff
@@ -337,6 +336,23 @@ export const CreateConnectorPage: React.FunctionComponent = () => {
     initPropertyValues();
   }, [connectorTypes]);
 
+  const ConnectionPropertiesError = ({ connectionPropsMsg }) => {
+    if(connectionPropsMsg.length !== 0){
+      return(
+        <ul>
+          {connectionPropsMsg.map((item, index) => (
+            <li key={index}>{item.property}: {item.message}</li>
```

Comment:
The display name of the property with error should be shown, so that the user can identify the problem property on the form